src/test/java/com/microsoft/sqlserver/jdbc/datatypes/SQLServerSpatialDatatypeTest.java

```diff
@@ -1264,6 +2184,16 @@ private void testWkt(String geoWKT, String geoWKTSS) throws SQLException {
         return data;
     }
 
+    private void validateGeometryGeography(String geoWKT, byte[] geomCLR, byte[] geogCLR,
+            byte[] geoWKB) throws SQLServerException {
+        Geometry geomWKT = Geometry.deserialize(geomCLR);
+        Geography geogWKT = Geography.deserialize(geogCLR);
+        assertEquals(geoWKT, geomWKT.asTextZM());
+        assertEquals(geoWKT, geogWKT.asTextZM());
+        assertTrue(Arrays.equals(geomWKT.STAsBinary(), geoWKB));
```

Comment:
Why can't you just remove assertTrue?

Reply:
No, the assertTrue is the part where the STAsBinary method (the one I'm adding) is getting tested, unless I misunderstood your question.